frontend/src/components/AddProductForm/AddProductForm.js

```diff
@@ -1,27 +1,27 @@
-import { useMutate } from "../../hooks/use-mutate"
-import { useRef } from "react"
+import { useMutate } from "../../hooks/use-mutate";
+import { useRef } from "react";
 
 const AddProductForm = () => {
-  const formRef = useRef()
-  const { addProduct } = useMutate()
+  const formRef = useRef();
+  const { addProduct } = useMutate();
 
   const handleSubmit = async (e) => {
-    e.preventDefault()
-    const formData = new FormData(formRef.current)
-    const data = Object.fromEntries(formData)
-    const input = { name: data.name, price: parseFloat(data.price)}
-    addProduct.mutate(input)
-    e.target.reset()
-  }
+    e.preventDefault();
+    const formData = new FormData(formRef.current);
+    const data = Object.fromEntries(formData);
+    const input = { name: data.name, price: parseFloat(data.price) };
+    addProduct.mutate(input);
+    e.target.reset();
+  };
 
   return (
     <>
       <form ref={formRef} onSubmit={handleSubmit}>
-        Name: <input type="name" name="name" required/>
-        Price: <input type="number" name="price" required/>
+        Name: <input type="name" name="name" required />
+        Price: <input type="number" name="price" required />
         <input type="submit" value="Submit" />
       </form>
     </>
-  )
-}
-export default AddProductForm
```

Comment:
Any reason this component was modified? I don't see any diffs besides lints

Reply:
I think I miss-clicked on it and when I saved it and my linter just added semicolons and spaces

Reply:
SGTM. You can restore it from main using
`git fetch
git restore -s origin/main-- frontend/src/components/AddProductForm/AddProductForm.js`

Commit the changes after restoring it and the file should disappear from your diffs.

Reply:
Perhaps we can edit the linter settings to not add semicolons, so our files can be consistent across

Reply:
I've added some settings that will help to keep everything a bit more consistent when using prettier in the README. @Eerian I suggest you install those then run the linter on the JS files you changed to keep the design language the same-ish

Reply:
ok this should be resolved now